generators/app/templates/src/redux/auth/actions.js

```diff
@@ -31,7 +31,7 @@ export const actionCreators = {
     await AuthService.removeCurrentUser();
     dispatch({ type: actions.LOGOUT, target: loginTarget });
     dispatch(
-      NavigationActions.reset({
+      StackActions.reset({
         index: 0,
         actions: [NavigationActions.navigate({ routeName: Routes.Login })]
```

Comment:
should you use StackActions here too?

Reply:
@ignaciosantise  `StackActions.reset` replace `NavigationActions.reset`, but `NavigationActions.navigate` and `NavigationActions.back` still exist